src/member/apps.py

```diff
@@ -3,3 +3,9 @@
 
 class MemberConfig(AppConfig):
     name = 'member'
+
+    def ready(self):
+        # import signal handlers
+        # (every models imported inside handlers will be instantiated as
+        # soon as the registry is fully populated.)
+        import member.signals.handlers  # noqa
```

Comment:
@lingxiaoyang your proposal could be accepted, we know that both options works good. But I think that we need to separate responsibilities. Maybe the `ready()` method does not have to know what are the necessary signals to be connected, all logic or all responsibility related to the receivers should be declared inside the signal python package.

Whatever is the final decision, I think that is good to make the corresponding example into [Django model project](https://gitlab.savoirfairelinux.com/django/modele).